encord/objects/coordinates.py

```diff
@@ -150,29 +152,92 @@ class Visibility(Flag):
     OCCLUDED = auto()
 
 
-@dataclass(frozen=True)
+@dataclass(init=False)
```

Comment:
This is fairly hacky. We have name, color, value stored in Skeleton Coordinate for no clear reason but it's definitely provided from BE. Because I am currently using SkeletonCoordinate both for the static data type used in labels and used to specify SkeletonTemplateCoordinates, one has to use hacky logic around people shouldn't have to specify the value, color, featureHash when creating elements however the FE strictly requires color, featureHash, value to be filled on. This schema isn't really well defined as far as I can tell